rotkehlchen/tasks/manager.py

```diff
@@ -554,6 +558,18 @@ def _maybe_update_curve_pools(self) -> Optional[list[gevent.Greenlet]]:
             tx_decoder=ethereum.transactions_decoder,
         )]
 
+    def _maybe_update_yearn_vaults(self) -> Optional[gevent.Greenlet]:
+        if should_update_protocol_cache(GeneralCacheType.YEARN_VAULTS) is True:
+            return self.greenlet_manager.spawn_and_track(
+                after_seconds=None,
+                task_name='Update yearn vaults',
+                exception_is_error=True,
+                method=self.query_yearn_vaults,
+                db=self.database,
+            )
+
+        return None
+
```

Comment:
No this will fail. Didn't you notice it when rebasing? This  module got modified. Check the code around.

```suggestion
    def _maybe_update_yearn_vaults(self) -> Optional[list[gevent.Greenlet]]:
        if should_update_protocol_cache(GeneralCacheType.YEARN_VAULTS) is True:
            return [self.greenlet_manager.spawn_and_track(
                after_seconds=None,
                task_name='Update yearn vaults',
                exception_is_error=True,
                method=self.query_yearn_vaults,
                db=self.database,
            )]

        return None
```

This is because task spawner can now return multiple greenlets.